### PR TITLE
add register

### PR DESCRIPTION
**Pull request by DMTRIP**

### CODE REVIEW

File: src/services/tourist-service.js [hunk 1]
```diff
@@ -22,8 +22,22 @@ class TouristService {
...
 
-    register(name, email, password) {
-        // return
+    async register({ nickName: nickname, ...other }) {
+          try {
+              const res = await request.post('register', {nickname, ...other});
+              localStorage.setItem('token', res.accessToken);
+              localStorage.setItem('refreshToken', res.refreshToken);
+          } catch (e) {
+
```
Comment (Dmitry528) [inline src/services/tourist-service.js:31]: Handle this error

File: src/components/app/app.js [hunk 2]
```diff
@@ -98,6 +99,7 @@ export default class App extends Component {
                   component={Profile} /> */}
```
Comment (Dmitry528) [inline src/components/app/app.js]: Delete comments

File: src/services/tourist-service.js (hunk 1 above)
Comment (DMTRIP) [reply src/services/tourist-service.js:31]: I handle it in register component      
const submit = async (values) => {
        try{
            await api.register(values);
            onAuth();
            setTimeout(() => setRedirect(true), 1000);
        } catch (e) {
            setErr(e);
        }
    };

File: src/components/app/app.js (hunk 2 above)
Comment (DMTRIP) [reply src/components/app/app.js]: deleted

**Review by Dmitry528 — APPROVED** [review]
Open server and client on your branch, check validation, check unique fields and its okay.
And create user, signIn in our application with my login and password
Open postgres and find this user, his - id === 60
Good work :)

**Review by andrii-kondratiuk — APPROVED** [review]

**Review by marrybbb — APPROVED** [review]

**Pull request merged**